Readme.md

```diff
@@ -0,0 +1,16 @@
+// do debouncing 
+// service worker generate NN need to use promises and localstorage 
+// MRU cache 
+// improve UI 
+
+Current problem that need to be addressed 
+To show output when the answer has been generated.Currently sending the callback to the controller which is using encryption task . Can promises be used if yes not able to figure out.
+
+
+/*******************/
+To run the code 
+
+npm run build 
+node server.js
```

Comment:
For client files, we should not be needing this just yet, especially since we are not using a framework yet